web/web.go

```diff
@@ -990,6 +990,15 @@ func incomingWebhook(c *api.Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	channelName := parsedRequest.ChannelName
+	webhookType := parsedRequest.Type
+
+	if parsedRequest.Attachments != nil {
```

Comment:
I wonder if we can/should set some high upper limit for the attachment size?  5mb or something?